### ISSUE TITLE
`set -o echo` causes error with `sh`

### ISSUE THREAD

**Issue by yurtesen** [OpenVPN/easy-rsa#361]
In certain circumstances (eg. with `ansible`) when I run `easyrsa` I get an error (v3.0.6) on Ubuntu 18.04
```
/easy-rsa/easyrsa3/easyrsa: 319: set: Illegal option -o echo
```

Also following gives the same error.
```
$ sh -c 'set -o echo'
sh: 1: set: Illegal option -o echo
```

Perhaps it is a bug?

**Comment by pschiffe** [comment]
This should be already fixed in the master, see #308 and #315 however, the new release hasn't been done yet, see #313

**Comment by yurtesen** [comment]
Sorry about the duplicate. I will close this report. Thanks for the quick response.
